release/README.md

```diff
@@ -61,6 +61,38 @@ Install the systemd services:
 Add a webhook to your GitHub project that calls http://host:8090/cockpit on
 "create" events (and nothing else!); set this in "Let me select individual events".
 
+# Deploying on OpenShift
+
+On any host, set up all necessary credentials as above, plus an extra
+`~/.config/github-webhook-token` with a shared secret.
+
+    release/build-secrets > /tmp/secrets.json
+
+to build a Kubernetes secret volume definition. Copy that to a machine that
+administers your OpenShift cluster, and deploy that secret volume and the other
+objects with
+
+    oc create -f /tmp/secrets.json
```

Comment:
I would like to suggest piping the secrets rather than storing yet another copy in a file. Can we do:

```
release/build-secrets | oc create -f -
```

Reply:
Nice icea. Done and pushed: https://github.com/martinpitt/cockpituous/blob/release-openshift/release/README.md#deploying-on-openshift